src/Interactive/Host/Interactive/Core/InteractiveHost.cs

```diff
@@ -170,10 +170,13 @@ private RemoteService TryStartProcess(string hostPath, CultureInfo culture, Canc
                         return null;
                     }
 
-                    lock (_outputGuard)
+                    Task.Run(() =>
                     {
-                        _output.WriteLine(InteractiveHostResources.Attempt_to_connect_to_process_Sharp_0_failed_retrying, newProcessId);
-                    }
+                        lock (_outputGuard)
+                        {
+                            _output.WriteLine(InteractiveHostResources.Attempt_to_connect_to_process_Sharp_0_failed_retrying, newProcessId);
+                        }
+                    });
```

Comment:
Perhaps we could add a method `WriteOutput(bool isError, string firstLine, string secondLine = null)` that does the Task.Run, so that we don't need to repeat the pattern mutliple times.